cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
It  fails with a timeout error on 5.7. (when the delay was set to 120). I ran it without the UI and it fails on Images

provider.validate_stats(ui=False)

Any ideas on what might be causing this? It keeps restarting the API, but the num_image() fails every time on 5.7. 
@psav @mfalesni @jkrocil